cmd/resolve-version/main.go

```diff
@@ -198,7 +201,7 @@ func matchReleaseSemver(releases []release, versionRequirement string) (matchRes
 	resolvedVersion := coll[len(coll)-1]
 
 	for _, rel := range filtered {
-		if rel.version.Equal(resolvedVersion) {
+		if rel.version.Equals(resolvedVersion) {
```

Comment:
I think the convention in stdlib is `Equal` (like `bytes.Equal`), but I already hate myself for even bringing this up

Reply:
That's inherited from the library I forked for this. I'll make a note to change that as I polish it up and document it :)